File: backend/src/main/java/com/kh/backend/model/vo/Memo.java
package com.kh.backend.model.vo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Memo {

	private int memoNo;
	private int userNo;
	private String content;
	private String targetDate; // 일단 String
}

File: admin-frontend/src/type/memo.ts
export interface Memo {
    memoNo: number,
    userNo: number,
    content: string,
    targetDate: string
}

export const initialMemo: Memo = {
    memoNo: 0,
    userNo: 10,
    content: '',
    targetDate: ''
}

export const initMemoList: Memo[] = [];